system-tests/__snapshots__/record_spec.js

```diff
@@ -132,8 +132,8 @@ Because this error occurred during a \`before each\` hook we are skipping the re
 
   (Uploaded Cloud Artifacts)
 
-  - Video - Done Uploading 1 kB 1/2 /XXX/XXX/XXX/cypress/videos/record_fail.cy.js.mp4
-  - Screenshot - Done Uploading 1 kB 2/2 /XXX/XXX/XXX/cypress/screenshots/record_fail.cy.js/record fails -- fails 1 -- before each hook (failed).png
+  - Video - Done Uploading 1 kB 1/2 in Xm, Ys ZZms /XXX/XXX/XXX/cypress/videos/record_fail.cy.js.mp4
```

Comment:
This may have already been discussed, but I think it makes more sense to have the time directly after the size:
`Done Uploading 1 kb in Xm, Ys ZZms 1/2`